src/ducks/mottatteOpplysninger/types.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Types
+ * ----------------------------------------------------------------------------------
+ * Dette er action types som benyttes for å indikere hvordan fagsaker skal manipuleres
+ * eller oppdateres.
+ */
```

Comment:
```suggestion
/**
 * Dette er action types som benyttes for å indikere hvordan fagsaker skal manipuleres
 * eller oppdateres.
 */
```

"Types" og --- er ikke nødvendig da navnet på filen er types :shrug: 

(Hvis du er uenig trenger du ikke å gjøre noe, har et personlig bias mot utkommentert kode og kommentarer som ikke gir verdi utenom dekorasjon :rofl: )

Reply:
Er ikke uenig, men tar det ikke i en PR som allerede er alt for stor. Får evt ta en opprydding en gang.